packages/libs/src/services/web3Manager/Web3Manager.ts

```diff
@@ -268,6 +268,14 @@ export class Web3Manager {
               bail(e)
               return
             }
+
+            if (e.response.status >= 500) {
```

Comment:
I think we can avoid touching it here and instead made the change here
/Users/ray/Documents/Audius/audius-protocol/packages/libs/src/sdk/services/DiscoveryNodeSelector/DiscoveryNodeSelector.ts

Reply:
I think Alec's point is that the above code doesn't actually use the mainline `post` wrapper linked here... but instead is using the `identityService.relay` function with a `baseURL` override.

Since there is a `retry` wrapper this will probably work, altho we might want to `throw` instead of `return` at line 276.

But yeah I think we could also change this around to have a more divergent code path for `discoveryRelay` that uses the DN post wrapper, and maybe even delete the identity relay code.  Possibly in a later PR if we want a quick fix.

Reply:
I see. game for this as a quick fix then

Reply:
Steve is right, that's why the post hook wasn't being called. I've pushed a change that does use the DN post wrapper though. This required chain_health being added to the health check as well.

Reply:
I can revert this change and submit it in a later PR if we are good with the quick fix.

Reply:
yes!